tests/e2e/specs/productDetailPage_spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Product detail page', () => {
+  before(() => {
+    cy.visit('/product/lemare-booties-0778-grey/M0E20000000E0WX');
+  });
+
+  it('Displays a single product details', () => {
+    cy.get('[data-test=pdp-product-title]', { timeout: 20000 })
+      .contains('Booties Lemare grey')
+      .parentsUntil('[data-test=product-description]')
+      .parent()
+      .find('[data-test=quickview-sku]')
+      .contains('M0E20000000E0WX')
+      .parentsUntil('[data-test=product-description]')
+      .parent()
```

Comment:
There are lots of moving up and down the DOM. Sometimes it's unavoidable (unfortunately) but I hope this isn't one of those cases. It would be easier to understand and maintain in the future if the test consisted of:
1) find the place where the info is
2) assert each of the info is correct
3) repeat until all the info is checked
If the path to find the place where the info is, is too complex, then it can be grouped under a `.then(($someInfo) => ...)` to avoid duplication of code.

So in short: try to improve this code 😄